### PR TITLE
VB bulk import

### PR DESCRIPTION
**Pull request by KillerX**
The basic idea is to re-use the master import but run it for each file (minus renaming the file).
This is currently not testable locally without a big refactor of several services that I started on but will not be able to complete to any sane extent before the end of the sprint so it will have to endure a production test.

### CODE REVIEW

File: workflows/ingest/masters.go [hunk 1]
```diff
@@ -55,91 +54,42 @@ func Masters(ctx workflow.Context, params MasterParams) (*MasterResult, error) {
 		return nil, err
 	}
 
-	if utils.IsMedia(result.Path.Local()) {
-		// This isn't run on VB masters in old system, but see no reason to not run it here.
-		result.AnalyzeResult, err = analyzeAudioAndSetMetadata(ctx, result.AssetID, result.Path)
```
Comment (andreasgangso) [inline workflows/ingest/masters.go:60]: did we lose this normalization now or is it happening somewhere else?

File: workflows/ingest/masters.go [hunk 2]
```diff
@@ -160,18 +110,84 @@ func uploadMaster(ctx workflow.Context, params MasterParams) (*MasterResult, err
...
+	}
+
+	importedVXs := map[string]paths.Path{}
+	errors := []error{}
+
+	for _, sourceFile := range sourceFiles {
+		file := params.OutputDir.Append(sourceFile.Base())
+
+		if filename != "" {
+			file = params.OutputDir.Append(filename)
```
Comment (andreasgangso) [inline workflows/ingest/masters.go:160]: How I'm reading this, we set filename only if `OrderFormOtherMaster, OrderFormVBMaster, OrderFormSeriesMaster, OrderFormLEDMaterial, OrderFormPodcast`.
And this condition right here looks like it is assuming sourceFiles will only have 1 file in that scenario, and with that assumption in mind we can safely do this (`file = params.OutputDir.Append(filename)`).
But it's hard to follow along on this and could have bugs if sourcefiles somehow got multiple items.

You could split the "bulk" scenario into a different function as part of the switch above, and split the shared code into functions instead.

**Review by andreasgangso — CHANGES_REQUESTED** [review]

File: workflows/ingest/masters.go (hunk 1 above)
Comment (KillerX) [reply workflows/ingest/masters.go:60]: This is not normalization, it's just analysis, and noone ever read that data in MB
